finagle-memcached/src/main/scala/com/twitter/finagle/memcached/util/ParserUtils.scala

```diff
@@ -44,7 +44,7 @@ object ParserUtils {
         segmentStart = segmentEnd + 1
       }
     }
-    split
+    split.toIndexedSeq
```

Comment:
Is this going to spit out a `Vector`? If so, that would be pretty bad.

Reply:
Yes. I pushed an approach in a separate commit that just `toSeq`'s this and changes the return type to Seq (which then propagates forward a bit). I don't see any immediate performance hazards for that.